torch/csrc/api/include/torch/nn/functional/fold.h

```diff
@@ -25,7 +25,7 @@ inline Tensor fold(const Tensor& input,
   } else {
     TORCH_CHECK(
         false,
-        "Input Error: Only 3D input Tensors are supported "
+        "Input Error: Only batched or unbatched 3D input Tensors are supported "
```

Comment:
Not so sure about how this error and docs updates should be phrased 🤔 

Thanks!

Reply:
Hm, maybe:
`Only unbatched (2D) or batched (3D) input Tensors are supported`

or even just:
`Only 2D or 3D input Tensors are supported`

wdyt?